curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-algorithmic-thinking-by-building-a-number-sorter/6621c5f02a5c272761db8661.md

```diff
@@ -0,0 +1,340 @@
+---
+id: 6621c5f02a5c272761db8661
+title: Step 8
+challengeType: 0
+dashedName: step-8
+---
+
+# --description--
+
+You should use `console.log()` to print out the result of `inputValues`. Write the code for this inside the `sortInputArray` function.
+
+Before going further, make sure to observe the data type of the printed result in the console. 
+
+To see the result for the `inputValues` array, log out the `inputValues` array to the console. Then click on the sort button and open up the console. You should see an array of strings like this:
+
+```js
+[ '8', '2', '4', '1', '3' ]
+```
+
+In the next step, you will convert those strings into numbers.
+
+# --hints--
+
+You should call `console.log()`.
+
+```js
+assert.match(code, /console\.log\(.*\);/);
```

Comment:
The semi-colon here should be optional. If a camper fails to use the semi-colon, the test will not pass, so the next hint and test won't be reached.
 
```suggestion
assert.match(code, /console\.log\(.*\);?/);
```